manifests/init.pp

```diff
@@ -320,10 +320,10 @@
 
   if $apache {
     $use_foreman_service = ! $passenger
-    $foreman_service_bind = '127.0.0.1'
+    $foreman_service_bind = 'unix:///run/puma.sock'
```

Comment:
One open question was whether this path sufficed or we should put it under `/run/httpd` or `/run/foreman`